src/main/java/org/folio/oaipmh/dao/InstancesDao.java

```diff
@@ -17,6 +20,19 @@ public interface InstancesDao {
 
   Future<RequestMetadataLb> getRequestMetadataByRequestId(String requestId, String tenantId);
 
+  /**
+   * Returns request metadata collection
+   */
```

Comment:
Optional. 
Since other endpoints don't have a javadoc, because their purpose is clear from the name, this javadoc can be also removed